native/chat/robotext-message.react.js

```diff
@@ -105,7 +105,15 @@ class RobotextMessage extends React.PureComponent<Props> {
         continue;
       }
       if (splitPart.charAt(0) !== '<') {
-        textParts.push(decodeURI(splitPart));
+        textParts.push(
+          <Markdown
+            key={robotextParts.indexOf(splitPart)}
+            darkStyle={true}
```

Comment:
Two things I'm not sure about: I've added key, since without it there was a yellow warning. I'm also passing `darkStyle={true}`, because we don't have any info about globalTheme here to pass proper value, but it's just a blue link on light/dark background and on both it looks good.
Let me know if there is a better solution about any of this!

Reply:
Yeah, you need a key here, since you're pushing a React element onto an array. Strings are their own keys, but React elements need explicit keys specified.

1. You need to get the theme data and set `darkStyle` based on that
2. Let's make sure that the key doesn't accidentally collide with the `id` keys used below. `id` keys are numeric, so we can solve this by prefixing a string like `text${robotextParts.indexOf(splitPart)}`
3. Let's avoid calling `indexOf` here, there's no need to search through the string for an index. We can initialize a `let keyIndex = 0;` variable at the start, and just use `text${keyIndex++}` as the key